frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -2049,11 +2050,11 @@ A component to render timestamp.<br/>The timestamps are synchronized between ind
 
 ---
 
-## `useModal`
+## `useOverlay`
 
 ### Summary 
 
-A hook to launch Modals.
+A hook to launch Overlays.
```

Comment:
I know the `useModal` description and others are terse, but I'd like to try to add more detail to these going forward and eventually fix existing APIs. Let's add more description here. Some questions that come to mind:

1. What is an overlay?
2. How do I position an overlay?
3. How do I close an overlay?
4. Can I launch multiple overlays?
5. Can I make an overlay modal?
6. What props if any can the component I pass in take?
7. What PF or SDK components can I use with overlays?

Reply:
Thanks @spadgett! I expanded the API docs with your suggestions. Could you please take another look?